devimint/src/main.rs

```diff
@@ -331,7 +331,7 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         .unwrap()
         .to_owned();
     assert_eq!(operation["operation_kind"].as_str().unwrap(), "wallet");
-    assert_eq!(operation["outcome"].as_str().unwrap(), "Claimed");
+    assert!(operation["outcome"]["Claimed"].as_object().is_some());
```

Comment:
**suggestion:** `assert_eq!(operation["outcome"].as_str(), Some("claimed"));`

But yeah, it's a backport so whateva.

Reply:
this is incorrect, default serialization of enums is very weird in serde.

see other comment